Organization.md

```diff
@@ -0,0 +1,102 @@
+# Organization
+
+Currently, the Distributed Text Services is not legally formalized as an organization (outside of the Github Organization and the Google Group).
+
+Although, the Distributed Text Services specifications is lead by a [Technical Committee](#technical_committee). It is responsible for the evolution and documentation of the specifications.
+
+
+## Technical Committee
+
+* The **Technical committee** is responsible for voting on feature requests and bugs
+* Once the initial committee is established, the committee itself is responsible for approving new members to the committee
```

Comment:
Having been established, the committee itself is now responsible for approving new members to the committee.